app/worker/smtp.go

```diff
@@ -142,3 +147,11 @@ func processSMTPTask(smtpTask *model.SMTPTask, job *model.Job,
 
 	return result, nil
 }
+
+func getEmailAddress(from string) string {
+	emailAddress, err := mail.ParseAddressList(from)
+	if err != nil || len(emailAddress) < 1 {
```

Comment:
is it possible that `emailAddress[0].Address` is empty? (if so, I would add a check as well.

Reply:
No, it is not possible (it would return an error)